script.js

```diff
@@ -21,6 +22,11 @@ let todos = [];
 
 let filterState = "all";
 
+let endIndex = 9;
+const todosNeedToBeLoaded = 6;
+let currentPage = 1;
+let totalPage = 1;
```

Comment:
magic number
A unique value with unexplained meaning or multiple occurrences which could (preferably) be replaced with a named constant.

```suggestion
import { INITIAL_PAGE, PAGE_LOAD_COUNT } from "./const.js"
let currentPage = INITIAL_PAGE;
let totalPage = INITIAL_PAGE;
const pageLoadCount = PAGE_LOAD_COUNT;
```


Do we need the endIndex? 
Can't endIndex be calculated everytime we apply pagination?

```js
const endIndex = currentPage * perPage
const arr = todos.slice(0, endIndex)
```
As slice takes care of the overflow, we don't need to think about it.

Reply:
bhai if we want to show same numbers of todos on both first page and after clicking the **Load More** , then we don't need the endIndex. But according to trello we have to show 9 todos at first, while clicking **Load More** another 6 todos will be added in the dom. So in this case I have handled that task differently.

Reply:
Got it. I think let's not make it complicated. Let's initially show 9 or `PER_PAGE_TODO` count and on each click of Load More button, we'll show `PER_PAGE_TODO` more.